src/common/injectGlobals.js

```diff
@@ -1,7 +1,9 @@
-import 'index.css';
 import queryString from 'querystring';
 
-localStorage.setItem('debug', process.env.REACT_APP_DEBUG || ''); // manually set because CRA doesn't allow arbitrary env variable names.
+const getApplicationEnvVar = envVarName =>
```

Comment:
This will choose the REACT_APP_* variable if available instead of the STORYBOOK_* variable. Is this intended? Do we want a command line arg in the run storybook script that could be used to select storybook instead?

Reply:
That was my intention. I didn't think of extending the script, no objection on such feature other than it's extra logic which I personally don't think we need. If you think that's a better approach, LMK ;)

Reply:
Can merge this without concern since the normal params take precedence... if you want to run your storybook with different values you could end up with conflict which would be frustrating. If that happens we can deal with it then.